### PR TITLE
[zh-cn]sync local-object-reference.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/kubernetes-api/common-definitions/local-object-reference.md

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/common-definitions/local-object-reference.md [hunk 1]
```diff
@@ -34,13 +34,10 @@ LocalObjectReference 包含足够的信息，可以让你在同一命名空间
...
 -->
 - **name** (string)
 
-  被引用者的名称。
-  更多信息: https://kubernetes.io/zh-cn/docs/concepts/overview/working-with-objects/names/#names。
-
-
-
-
+  被引用者的名称。该字段实际上是必需的，但由于向后兼容性允许为空，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/common-definitions/local-object-reference.md:41]:
```suggestion
  被引用者的名称。该字段实际上是必需的，但由于向后兼容性允许为空。
```

File: content/zh-cn/docs/reference/kubernetes-api/common-definitions/local-object-reference.md [hunk 2]
```diff
@@ -34,13 +34,10 @@ LocalObjectReference 包含足够的信息，可以让你在同一命名空间
...
+  此处具有空值的这种类型的实例几乎肯定是错误的。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/common-definitions/local-object-reference.md:42]:
```suggestion
  这种类型的实例如果此处具有空值，几乎肯定是错误的。
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**